gallery/visualtests/pageLayouts.js

```diff
@@ -113,7 +113,7 @@ describe('Page Layout', function() {
 
       it('passes a11y checks', async function() {
         // This rule appears to not work correctly on elements that are partially scrolled off the page
-        await a11yTest(builder => builder.disableRules('color-contrast'));
+        await a11yTest(builder => builder.disableRules('color-contrast'), true);
```

Comment:
This builder customization is no longer necessary since `a11yTest` does the same thing by default.
```suggestion
        await a11yTest(null, true);
```
Also remove the comment above, and make equivalent changes in the other a11y test below.

Reply:
It is updated here ff4855eca50726eb77872778ee9edd74cc7836be.